package.json

```diff
@@ -52,15 +52,15 @@
         "chai": "^4.3.4",
         "coveralls-next": "^4.2.0",
         "dedent": "^0.7.0",
-        "eslint": "8.0.1",
-        "mocha": "^9.1.3",
+        "eslint": "^8.57.0",
+        "mocha": "^10.3.0",
         "nyc": "^15.1.0",
         "q": "^1.5.1",
-        "rimraf": "^2.6.2",
-        "sinon": "^11.1.2",
-        "source-map-support": "^0.5.13",
-        "ts-node": "^10.3.1",
-        "typescript": "^4.4.4"
+        "rimraf": "^5.0.5",
```

Comment:
Looks like rimraf is failing the build. Perhaps we need to roll back to rimraf v4?

Reply:
Actually I had to rollback rimraf to v3 AND typescript to v4 in order to stop getting errors with node 12.22.7

Reply:
resolved by upgradign node to 14.17